src/index.js

```diff
@@ -219,6 +219,12 @@ exports.createFromJSON = function (obj, callback) {
   }
 }
 
+exports.isPeerId = function (peerId) {
+  if (peerId.constructor && peerId.constructor.name) {
+    return peerId.constructor.name === 'PeerId'
```

Comment:
that's the point of symbols, you reuse the same symbol on all instances as you can see above in my example